prover-server/src/prove.rs

```diff
@@ -124,6 +122,7 @@ pub fn write_agg_proof(dir: &Path, proof: &AggCircuitProof) {
     proof.write_to_dir(&mut proof_path);
 }
 
+#[allow(dead_code)]
```

Comment:
Do we need this?

Reply:
I think it's redundant code. I decided to remove it.